test/test_integration_credentials.py

```diff
@@ -248,58 +256,101 @@ def test_https_logout_twice_returns_could_not_logout_message(self):
                                               ['logout'], exit_code=1)
 
         self.assertIn("Failed to log out. You are already logged out", unsuccessful_logout.strip())
-        with open(DEFAULT_NETRC_FILE) as netrc_file:
-            assert netrc_file.read().strip() == "", 'netrc file is not empty!'
+
+    '''
+        Validates when a user can logout successfully
+    '''
 
     @integration_test(True)
-    def test_no_netrc_and_logout_returns_successful_logout_message(self):
-        try:
-            os.remove(DEFAULT_NETRC_FILE)
-        except OSError:
-            pass
-        logout = self.invoke_cli(self.cli_auth_params,
-                                              ['logout'], exit_code=1)
-        self.assertIn("You are already logged out", logout.strip())
+    def test_https_logout_successful(self):
+
+        self.invoke_cli(self.cli_auth_params,
+                        ['login', '-i', 'admin', '-p', self.client_params.env_api_key])
+
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['logout'])
+
+        self.assertIn('Successfully logged out from Conjur', output.strip())
 
     '''
-    Validates logout doesn't remove another entry not associated with Cyberark
+        Validates that if a user configures the CLI in insecure mode and runs the command not in 
+        insecure mode, then they will fail
     '''
-    @integration_test(True)
-    def test_https_netrc_does_not_remove_irrelevant_entry(self):
 
-        with open(f"{DEFAULT_NETRC_FILE}", "w") as netrc_test:
-            netrc_test.write(f"machine {self.client_params.hostname}\n")
-            netrc_test.write("login admin\n")
-            netrc_test.write(f"password {self.client_params.env_api_key}\n")
+    @integration_test(True)
+    def test_cli_configured_in_insecure_mode_but_run_in_secure_mode_raises_error(self):
```

Comment:
this test appears three times here and twice in netrc. 
1. Do we need them three times? It looks like it does the same thing..
2. The names of the functions should be different between the two files otherwise they might cancel each other out or the only one will be run